xgboost_ray/main.py

```diff
@@ -599,6 +602,8 @@ def load_data(self, data: RayDMatrix):
         else:
             self._local_n[data] = len(param["data"])
 
+        # set nthread for dmatrix conversion
+        param["nthread"] = int(_ray_get_actor_cpus())
```

Comment:
this is going to fail if `_ray_get_actor_cpus` returns `None` (as that cannot be converted to int)